yang-module-versioning/draft-ietf-netmod-yang-module-versioning.xml

```diff
@@ -169,8 +165,9 @@
   <section title="Updating a YANG module with a new revision" anchor="update_rules">
     <t>This section updates <xref target="RFC7950"/> section 11 and <xref target="RFC6020"/> section 10 to refine the rules for permissible changes when a new
     YANG module revision is created.</t>
-    <t>A new module revision MAY contain NBC changes, e.g., the semantics of
-    an existing data-node definition MAY be changed in an NBC manner without
+    <t>While a new module revision SHOULD NOT contain NBC changes, they are allowed.
+    For example, the semantics of
+    an existing data-node definition can change in an NBC manner without
     requiring a new data-node definition with a new identifier. A YANG extension,
```

Comment:
I would perhaps suggest wording this the other way around, e.g., perhaps something like ...

```New module revisions SHOULD only contain changes that are backwards compatible (as per the rules in 3.1.1) to the previous revision.  This maximizes backwards compatibility for clients.  However, NBC changes MAY be made:
 (i) For bugfixes, particularly where the likely client impact is low
 (ii) To mark definitions as obsolete (or remove them), after a suitable deprecation period.
 (iii) To refine new and unstable modules, or new and unstable definitions within otherwise stable modules.  The modules should contain clear guidance to indicate which parts of the module are regarded as unstable.
 (iv) Where the module needs to be restructured, e.g., to add new functionality, and the cost of adding the new functionality in a backwards compatible way is disproportionate to the expected benefits of greater client backwards compatibility.
```

Reply:
Thanks.  I folded in your changes to a degree.  The bit about indicating which part of the module was unstable felt out of place here.